.github/workflows/release.yml

```diff
@@ -21,6 +21,13 @@ jobs:
     runs-on: ubuntu-latest
     name: "Release a new version"
     steps:
+        - id: killswitch
+          if: ${{ !github.event.inputs.force }}
+          run: |
+            if curl -s "https://api.github.com/repos/$GITHUB_REPOSITORY/issues?state=open&labels=release-blocker" | grep -Pzvo '\[[\s\n\r]*\]'; then
+              echo "Open release-blocking issues found, cancelling release...";
+              curl -sf -X POST -H 'Accept: application/vnd.github.v3+json' -H 'Authorization: token ${{ secrets.GITHUB_TOKEN }}' https://api.github.com/repos/$GITHUB_REPOSITORY/actions/runs/${{ github.run_id }}/cancel;
```

Comment:
What happens if this fails? Does the process continue ruthlessly or does the whole workflow fail ?

Reply:
Unfortunately, the workflow tries to keep going but it will report as a failure (due to the `-f` flag set on the `curl` call).

Honestly, I'm really waiting/wishing for GitHub to introduce a saner and simpler way to fail all subsequent steps in a workflow but we'll see.

Reply:
@fpacifici also see this: https://github.com/getsentry/relay/pull/714#discussion_r471467736

Reply:
Well, damn. I guess this is only an issue when if the cancel call fails. 
Is there any reason to alert people that it happened so that releases are not silently created even with open release-blocking issues ?

Reply:
@fpacifici yeah, we'd get an email saying the workflow failed as this call failing would cause this step/job to fail (due to `curl -f`). Interestingly, since nothing else relies on this step running, other steps would keep going. The alternative is this step outputting a value and then manually adding an `if: '{{ variable }}'` to all other steps. Even weirder, in that case the workflow would succeed as the ifs will skip the steps but won't cause an error.

I'd say this is acceptable risk at the moment.

Reply:
Well, it seems there is no way to make this safer and it only fails when we fail at canceling, so I guess we can take it.